util/latexonline

```diff
@@ -113,12 +117,23 @@ if [[ $GIT_TRACKED == true ]]; then
 else
     supportingFiles=$@
 fi
-tar -cj $target $supportingFiles > $tarball
 
 # create tmp file for headers
 dumpHeaders=`mktemp latexCurlHeaders-XXXXXX`
 outputFile=`mktemp latexCurlOutput-XXXXXX`
-curl -L --post301 --post302 --post303 -D $dumpHeaders -F file=@$tarball "$host/data?target=$target&force=$FORCE&command=$command" > $outputFile
+
+if [[ $GIT_REPO == true ]]; then
+  GIT_URL=$(git remote get-url origin)
```

Comment:
I assume we should use whatever is the current upstream for the tracking branch, instead of the `origin`.